lessons/specifics/guardian.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: page
+title: Guardian (Basics)
+category: specifics
```

Comment:
@hassox I'll take a peek, there may be a couple other small config changes necessary for this new section.